Description: Prefer dynamically linked binary for perseustest
Author: Jakob Ketterl <jakob.ketterl@gmx.de>

* link perseustest dynamically.
Index: libperseus-sdr-debian/examples/Makefile.am
===================================================================
--- libperseus-sdr-debian.orig/examples/Makefile.am
+++ libperseus-sdr-debian/examples/Makefile.am
@@ -25,14 +25,11 @@ AM_LDFLAGS = -ldl
 endif
 
 
-bin_PROGRAMS = perseustest_dyn perseustest
+bin_PROGRAMS = perseustest
 check_PROGRAMS = simple
 
 perseustest_SOURCES = perseustest.c fifo.c fifo.h $(top_srcdir)/perseus-sdr.h
-perseustest_LDADD   = ../libperseus-sdr.a
-
-perseustest_dyn_SOURCES = perseustest.c fifo.c fifo.h $(top_srcdir)/perseus-sdr.h
-perseustest_dyn_LDADD   = ../libperseus-sdr.la
+perseustest_LDADD   = ../libperseus-sdr.la
 
 simple_SOURCES = simple.c $(top_srcdir)/perseus-sdr.h
 simple_LDADD   = ../libperseus-sdr.la
